vaultSDK/README.md

```diff
@@ -0,0 +1,9 @@
+# vaultSDK
+
+Dart bindings for the vaultSDK
+
+## Getting Started
+
+Place all the header files from `vdk/include` in `ios/Classes/`
+
+Place the `android_arm64` and `win_x64` folders from `vdk/lib/` into a new folder `android/src/main/jniLibs`, and rename them to `arm64-v8a` and `x86_64` respectively.
```

Comment:
This should be `android_arm64` and `android_x64`.

Reply:
Should new folder names remain the same? so `android_arm64 -> arm64-v8a` and `android_x64 -> x86_64`

Reply:
Yes, those are the correct ABI names.